sandbox/src/mocks/r4/NHSDigital-OperationOutcome.json

```diff
@@ -1,21 +1,24 @@
 {
```

Comment:
Should this have HTTP code in the name like the others? i.e. 400?

Reply:
This is used in existing R4 endpoints. With [ERSSUP-63459](https://nhsd-jira.digital.nhs.uk/browse/ERSSUP-63459) I assume a commit will go into develop to replace/change this, when we fix the OAS so they return an operation outcome. In the meantime I've left as is.